### ISSUE TITLE
Check for update error

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3502]
**Reported by srippon on 2013-09-05 04:13**
Using NVDA (2013.1.1) on a Windows 7 PC.  When clicked the NVDA icon in the system tray and selected Help > Check for update... I received the following error:

Error

Error checking for update.
[OK]

-end-

Could NVDA please provide a more descriptive error message to explain why it couldn't check for an update?  

I believe the error may have been because I do not have administrator rights on this work computer.  Can NVDA update itself under a user account that doesn't have administrator rights?

Thanks in advance.

**Comment by jcsteh** [comment]
Unfortunately, this log doesn't reveal anything useful and we no longer have contact with the reporter. Closing.
